### DISCUSSION TITLE
Feature request: support jumping to lines in files

### DISCUSSION THREAD

**Discussion by keith** [community/community#5] [Mobile]
When tapping on a link such as https://github.com/llvm/llvm-project/blob/f5314d15af4f4514103ea12c74cb208538b8bef5/clang/tools/driver/driver.cpp#L79-L109 the correct file is opened, but the lines parameter seems to be ignored. It would be great if it would jump to the right lines, and highlight them

**Comment by candyho** [comment]
Thanks for flagging this. We're keeping track of the request to support jump to lines in files. Will surface this to the team again!
